src/components/_modal/pages/order/utils/order.utils.js

```diff
@@ -9,12 +9,11 @@ import styles from "@/components/_modal/pages/order/Order.module.css";
  * @param {function} setMail
  */
 export function onMailChange(value, valid, updateLoanerInfo, setMail) {
-  valid &&
-    updateLoanerInfo &&
-    updateLoanerInfo({ userParameters: { userMail: value } });
+  valid?.status && updateLoanerInfo && console.log("updating");
```

Comment:
Der er en console.log for meget, og du opdaterer kører `updateLoanerInfo({ userParameters: { userMail: value } });` hver gang `onMailChange` kører, også uden at `valid?.status` og `updateLoanerInfo` er blevet tjekket :thinking:

Reply:
hov - væk med den